src/main/resources/native/android/dalvik/source/MainActivity.java

```diff
@@ -235,13 +235,13 @@ public boolean dispatchTouchEvent(MotionEvent event) {
                     for (int i = 0; i < pcount; i++) {
                         actions[i] = pointerIndex == i ? actionCode : ACTION_POINTER_STILL;
                         ids[i] = event.getPointerId(i);
-                        touchXs[i] = (int) (event.getX(i)/density);
```

Comment:
this requires a conceptual change in OpenJFX as well. Is there a corresponding issue?

Reply:
Reverted